docs/rules/bugs/leaked-internal-reference.md

```diff
@@ -44,6 +45,13 @@ Do note that if you disagree with this rule, you don't need to disable it unless
 something else. If you don't use underscore prefixes, nothing will be reported by this rule anyway. It does however
 mean that the benefits listed above won't apply to your project.
 
+## Exceptions
+
+In test files and packages, it can be useful to reference internal rules and
+functions to ensure good test coverage. If you want to run this rule for test
+files too, you can set `include-test-files: true` in the configuration for this
```

Comment:
It isn't super clear here that this rule is disabled by default for tests. I'd put more emphasis on that, given that it's probably the only rule where that is the case.

Reply:
Yeah that's fair, I have updated in 1abaa961d24e6a312dc6e9a1b5d25f6da02c9a19